app/src/main/java/org/exoplatform/fragment/PlatformWebViewFragment.java

```diff
@@ -226,7 +226,7 @@ public boolean onCreateWindow(WebView view, boolean isDialog, boolean isUserGest
           @Override
           public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request) {
             String url = request.getUrl().toString();
-            if (url.contains("/jitsi/meet")) {
+            if (url.contains("/jitsi/meet") || url.startsWith("https://accounts.google.com")) {
```

Comment:
Could we be more specific with the needed Google drive URL, the one used here is the one for login.
Besides, the problem is just when we want to connect for the first time or is it persistent when we did already enabled the integration with GDrive and we want to select another document

Reply:
- The problem is related to the external links, and how close the window after the process ended because the JS window.close() was not working on webview so im update the code according to that.
- The jitsi call also related to this update, so no need to the back button after the call is finished.